marketplacetests/tests/test_marketplace_incorrect_pin.py

```diff
@@ -0,0 +1,44 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from fxapom.fxapom import FxATestAccount
+
+from marketplacetests.marketplace_gaia_test import MarketplaceGaiaTestCase
+from marketplacetests.marketplace.app import Marketplace
+from marketplacetests.payment.app import Payment
+
+
+class TestMarketplaceIncorrectPin(MarketplaceGaiaTestCase):
+
+    def test_incorrect_pin(self):
+
+        APP_NAME = 'Test Zippy With Me'
+        good_pin = '1234'
+        bad_pin = '1111'
```

Comment:
I'm not sure that I agree that creating a dictionary and then having to reference the dictionary throughout the test is better (i.e., we'd have to use `test_data['app_name']` throughout the test rather than simply `app_name`), but I agree with your casing comment. I think `APP_NAME` is left over from when they were named like constants, but that doesn't really make sense. I will correct the case for `APP_NAME`.